### PR TITLE
Moderators can edit tag descriptions

### PR DESCRIPTION
**Pull request by robbiecarlton**

### CODE REVIEW

**Comment by robbiecarlton** [comment]
screen shot 2017-02-13 at 6 31 22 pm

**Comment by robbiecarlton** [comment]
screen shot 2017-02-13 at 6 33 32 pm

File: src/reducers/tags.js [hunk 1]
```diff
@@ -48,7 +49,29 @@ const decrementForRemovedTag = (state, { type, error, meta }) => {
 
 export const tagsByQuery = composeReducers(
   appendPayloadByPath(FETCH_TAGS, 'meta.cache.id', 'items', t => t.id),
-  filterOutRemovedTag
+  filterOutRemovedTag,
+  (state, { type, meta }) => {
```
Comment (robbiecarlton) [inline src/reducers/tags.js:53]: @levity - this is what I was asking about. I'm not sure why we have tagsByCommunity and tagsByQuery both containing tag data. makes sense to pattern it after posts, and have tagsByCommunity contain the data and tagsByQuery just the ids.

But I didn't want to go further down that direction without checking with you.

I can also just leave it as is if you think there's higher priorities.

**Comment by robbiecarlton** [comment]
@levity - this is just missing tests, but I'm holding off on them until we've decided what to do with the reducers. See my comment in reducers/tags/js

**Comment by levity** [comment]
looking good so far, just add tests

**Comment by robbiecarlton** [comment]
@levity - tests are there, good to merge?

File: test/components/TagSettings/component0.test.js [hunk 2]
```diff
@@ -0,0 +1,188 @@
+import '../../support'
```
Comment (levity) [inline test/components/TagSettings/component0.test.js]: i think this was an accidental file copy

File: src/components/CommunityMenu.js [hunk 3]
```diff
@@ -3,7 +3,7 @@ import cx from 'classnames'
 import { nextPath } from '../util/navigation'
 import { merge } from 'lodash'
 import { flow, get, map, sortBy } from 'lodash/fp'
-import { IndexA, A } from './A'
+import A, { IndexA} from './A'
```
Comment (levity) [inline src/components/CommunityMenu.js]: space before }

**Review by levity — DISMISSED** [review]
just a bit of cleanup & you're good to merge

File: test/components/TagSettings/component0.test.js (hunk 2 above)
Comment (levity) [reply test/components/TagSettings/component0.test.js]: oh you already got it, nice :)

**Pull request merged**